README.md

```diff
@@ -253,18 +253,18 @@ function* treeWalker(refresh) {
           // object.
           defaultHeight: 30,
           id,
-          isLeaf: children.length === 0,
+          isLeaf: children?.length === 0,
```

Comment:
The same situation as in the previous comment.

Reply:
```suggestion
          isLeaf: !children || children.length === 0,
```